templates/web/task_detail.html

```diff
@@ -67,6 +67,7 @@ <h6>
         {{ submit.created_at|timeuntil:submit.assignment.deadline }} after deadline
     </span>
     {% endif %}
+    {% if submit.ip_address %}(from {{ submit.ip_address }}){% endif %}
```

Comment:
I would not show this by default here, or at least not to students. Please show it only for teachers.

Reply:
I suggest to show only, let's say, first 7 digits of address's SHA256. That should be enough for the comparison.